### PR TITLE
[SKIP CI] Update the vDVS documentation with steps to run tests on Windows VM

### PR DESCRIPTION
**Pull request by ashahi1**
Updated contributing.md with steps to run windows test
```
      1. scp ~/.ssh/id_rsa.pub root@<Windows-VM-IP>:C:/Users/root/.ssh/authorized_keys
      2. export WIN_VM1=<Windows-VM-IP>
      3. make build-windows-plugin 
      4. make deploy-windows-plugin
      5. make test-e2e-windows
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -277,6 +277,15 @@ Add a multi-string key named Environment and add the following lines to it.
     GOPATH=C:\Users\root\go\
     PATH=C:\Program Files\mingw64\bin\;C:\Perl64\bin;C:\Python27;C:\Program Files (x86)\GnuWin32\bin;C:\Program Files\7-Zip;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Users\Administrator\AppData\Local\Microsoft\WindowsApps;C:\Go\bin;C:\Program Files\Git\cmd;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files\Docker;
 ```
+### To run tests on Windows VM
```
Comment (shuklanirdesh82) [inline CONTRIBUTING.md]: following steps are not only for running test in fact those are for building, deploying and testing (of course) ... this title should be generic enough.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -277,6 +277,15 @@ Add a multi-string key named Environment and add the following lines to it.
...
+
+    Following steps need to be executed to run tests on windows VM:
+
+      1. scp ~/.ssh/id_rsa.pub root@<Windows-VM-IP>:C:/Users/root/.ssh/authorized_keys
```
Comment (shuklanirdesh82) [inline CONTRIBUTING.md]: Inline comment or one line description should be provided for the better understanding about each command what it is significant for .

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -277,15 +277,21 @@ Add a multi-string key named Environment and add the following lines to it.
...
     PATH=C:\Program Files\mingw64\bin\;C:\Perl64\bin;C:\Python27;C:\Program Files (x86)\GnuWin32\bin;C:\Program Files\7-Zip;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Users\Administrator\AppData\Local\Microsoft\WindowsApps;C:\Go\bin;C:\Program Files\Git\cmd;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files\Docker;
 ```
-### To run tests on Windows VM
+### Build and deploy the plugin and run tests on Windows VM
 
-    Following steps need to be executed to run tests on windows VM:
+```
+      # setup the ssh key on Windows VM:
+      scp ~/.ssh/id_rsa.pub root@<Windows-VM-IP>:C:/Users/root/.ssh/authorized_keys
+      # set the environment variable:
```
Comment (shuklanirdesh82) [inline CONTRIBUTING.md]: nit: need an empty line to separate out each steps

**Comment by ashahi1** [comment]
Addressed all the comments. Merging the changes.

**Pull request merged**